crypto-macros/src/entity_derive_new/derive_impl.rs

```diff
@@ -0,0 +1,339 @@
+use proc_macro2::TokenStream;
+use quote::quote;
+
+use crate::entity_derive_new::Entity;
+
+impl quote::ToTokens for Entity {
+    fn to_tokens(&self, tokens: &mut TokenStream) {
+        let impl_entity_base = self.impl_entity_base();
+        let impl_entity_generic = self.impl_entity_generic();
+        let impl_entity_wasm = self.impl_entity_wasm();
+        let impl_borrow_primary_key = self.impl_borrow_primary_key();
+
+        // let entity_transaction_ext_impl = self.entity_transaction_ext_impl();
+        tokens.extend(quote! {
+            #impl_entity_base
+            #impl_entity_generic
+            #impl_entity_wasm
+            #impl_borrow_primary_key
+        });
```

Comment:
This could be written as
```rust
        tokens.extend([
            impl_entity_base,
            impl_entity_generic,
            impl_entity_wasm,
            impl_borrow_primary_key
        ]);
```
and avoid the `quote!`. Or, even better:
```
        tokens.extend(self.impl_entity_base());
        tokens.extend(self.impl_entity_generic());
...
```